stack/docker-compose.yml

```
@@ -49,6 +51,10 @@ services:
         chmod 600 /tmp/pgpass;
         /entrypoint.sh
       "
+    healthcheck:
+      <<: *healthcheck
+      test: [ "CMD", "wget", "-O", "-", "http://localhost:5050/misc/ping" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)